### ISSUE TITLE
Player controls don't appear on Opensea on mobile

### ISSUE THREAD

**Issue by chrishobcroft** [NFTsTv/Broadcastr#41]
I load up e.g. [this livestream NFT](https://opensea.io/assets/matic/0x84af56daf2eff8c1c76c636b0ee0e853ee777a46/1) in Firefox/DuckDuckGo browser on Android.

I tap to view a recorded stream, which opens successfully. It also successfully loads

But then (unexpected behaviour), I cannot unmute, and the player controls are unavailable to me. I can tap the video to pause/play, but that's all. The same happens with live content.

I expect to be able to unmute in all use cases - this is non-negotiable.
I don't need to be able to skip fwd / back for the livestream (not until there is ability to skip back).
I also don't need to be able to skip fwd / back for recorded content, this is only a "nice to have".

Works fine on Rarible and NFTScan:
![image](https://user-images.githubusercontent.com/2212651/219375704-0a03e897-7324-49b4-b480-ba68c860f2e3.png)
![image](https://user-images.githubusercontent.com/2212651/219375729-cb1c9b4d-a82d-439a-ba9b-1db991a7d766.png)

It is probably an issue that OpenSea will need to fix, but maybe we can help them (and others) to fix things, so that we can distribute content EVERYWHERE lol
